integration-tests/gradle/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.9-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.8-bin.zip
```

Comment:
Just spotted this after my review. I must say I'm confused:
```
find . -name gradle-wrapper.properties | xargs grep distributionUrl
./integration-tests/gradle/gradle/wrapper/gradle-wrapper.properties:distributionUrl=https\://services.gradle.org/distributions/gradle-6.9-all.zip
./devtools/project-core-extension-codestarts/target/classes/gradle-wrapper/gradle/wrapper/gradle-wrapper.properties:distributionUrl=https\://services.gradle.org/distributions/gradle-6.9-bin.zip
./devtools/gradle/gradle/wrapper/gradle-wrapper.properties:distributionUrl=https\://services.gradle.org/distributions/gradle-6.9-all.zip
./devtools/platform-descriptor-json/target/classes/gradle-wrapper/gradle/wrapper/gradle-wrapper.properties:distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.3-bin.zip
```
(this is from `main`)

Reply:
oh nice catch, I will revert that file